package.json

```diff
@@ -71,18 +71,29 @@
     "system-css": "^0.1.0",
     "system-text": "^0.1.0",
     "systemjs-builder": "^0.15.16",
-    "typescript": "~1.8.2",
-    "typings": "~1.0.4"
+    "typescript": "~2.0.3"
   },
   "dependencies": {
     "@angular/common": "~2.0.0",
     "@angular/compiler": "~2.0.0",
-    "@angular/core": "~2.0.0",
+    "@angular/core": "^2.0.1",
```

Comment:
Can we keep this as a '~' instead of a '^'? Otherwise npm will pull in angular 2.1.0 without any explicit change.